### PR TITLE
Create CountButton and SearchBar.

### PR DESCRIPTION
**Pull request by willrlin**
Add some of the logic from the tutorial to get some fake products and show them under a SearchBar.

### CODE REVIEW

**Comment by willrlin** [comment]
Elen/Abbie could you review this pull request?

File: reactsite/src/App.js [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      {products.length > 0 ? <SearchBar products={products} /> : "Loading..."}
+      {/* <CountButton incrementBy={1} buttonColor={"blue"} /> */}
```
Comment (elenm) [inline reactsite/src/App.js]: Should we delete this line?

**Review by elenm — APPROVED** [review]
Which tutorial are you using for this? Curious to take a look and follow along.

File: reactsite/src/App.js (hunk 1 above)
Comment (willrlin) [reply reactsite/src/App.js]: Done!

**Review by abbiefarr — APPROVED** [review]

**Pull request merged**